core-processor/src/processing.rs

```diff
@@ -256,10 +257,15 @@ fn process_error(
         let (err, err_payload) = case.to_reason_and_payload();
 
         // Panic is impossible, unless error message is too large or [Payload] max size is too small.
-        let err_payload = err_payload
-            .into_bytes()
-            .try_into()
-            .unwrap_or_else(|_| unreachable!("Error message is too large"));
+        let err_payload = err_payload.into_bytes().try_into().unwrap_or_else(|_| {
+            let err_msg =
+                format!("process_error: Error message is too big. Message id - {message_id}, error payload - {}",
+                case.to_reason_and_payload().1
+            );
+
+            log::error!("{err_msg}");
+            unreachable!("{err_msg}")
+        });
```

Comment:
can do something similar to here
https://github.com/gear-tech/gear/blob/e917669ef0079f8e54fa143f446275a56eb96e61/pallets/gear/src/manager/task.rs#L160

Reply:
Maybe I can use special `format` that looks like binding.

```rust
format!("{value}", value=4); 
```

Reply:
I meant the error message structure used under the ref I sent

Reply:
Got you, `try_into` consumes `Vec<u8>` by self here:

```rust
let err_payload = err_payload.into_bytes().try_into().unwrap_or_else(|_| {
```
So to prevent double clone, I have create new copy in error handler code.

Reply:
Nvm, fixed